drivers/led/Kconfig.gpio

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2018 Manivannan Sadhasivam
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config LED_GPIO
+	bool "LED GPIO driver"
+	depends on GPIO
+	help
+	  Enable software driven LED using GPIO pin.
+
+# ---------- LED 0 ----------
+
+config LED0_GPIO
```

Comment:
I'm finding 'LED0_GPIO' a bit confusing, as I read it 'GPIO of LED0'.
Would LED_GPIO_0 be a bit less confusing?

Reply:
Actually, the naming convention aligns with the DTS generated header (LEDx_GPIO). But anyway, `LED_GPIO_x` seems to be good.

Reply:
@Mani-Sadhasivam , sorry but I still find it a bit confusing actually. What about GPIO_LED ?
@MaureenHelm , any opinion on that?